core/service/round.ts

```diff
@@ -337,6 +334,12 @@ class RoundService implements IRoundService {
     }
 
     public async apply(round: Round = RoundRepository.getCurrentRound()): Promise<void> {
+        const prevRound = RoundRepository.getPrevRound();
+        if (prevRound && prevRound.startHeight) {
+            prevRound.endHeight = round.startHeight - 1;
```

Comment:
We need to set real `endHeight` from the last block. We will have the wrong `endHeight` if in the round will not be forged block.